runhttpd.sh

```diff
@@ -18,6 +18,17 @@ sed -i -e 's|\(^[[:space:]]*\)\(DocumentRoot\)\(.*\)|\1\2 "/shared/html"|' \
     -e 's|<Directory "/var/www/html">|<Directory "/shared/html">|' \
     -e 's|<Directory "/var/www">|<Directory "/shared">|' /etc/httpd/conf/httpd.conf
 
+# Remove log files from last deployment
+if [ -d "/shared/log/httpd" ]; then
```

Comment:
This check is not needed, `-f` flag does it for you. You can do just
```bash
rm -rf /shared/log/httpd
mkdir -p /shared/log/httpd
```

Reply:
Fixed.